docs/docs/README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Support disclaimer
 
-!!! warning "Officially support warning"
```

Comment:
We should update https://github.com/atlassian-labs/data-center-terraform/blob/ff1a0f6136ced65e62fe38bae3fccc06f3027c5c/docs/docs/troubleshooting/SUPPORT_BOUNDARIES.md?plain=1#L4 as well.